troposphere/awslambda.py

```diff
@@ -63,6 +63,7 @@ class Function(AWSObject):
         'Runtime': (basestring, True),
         'Timeout': (positive_integer, False),
         'VpcConfig': (VPCConfig, False),
+        'FunctionName': (basestring, False),
```

Comment:
It would be good to alphabetize this in the props and then force push the change back out. Might also be good to change the commit to be "Add FunctionName to Lambda::Function #450". Thanks.